### PR TITLE
7-arrays_elena-ambitska

### PR DESCRIPTION
**Pull request by elena-ambitska**

### CODE REVIEW

File: homeworks/olena.ambitska_elena-ambitska/7-arrays-loops/arrays.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
+const arrayOne = [3, 0, -5, 1, 44, -12, 3, 0, 0, 1, 2, -3, -3, 2, 1, 4, -2, -3, -1];
+const arrayTwo = [-1, -8, -2];
+const arrayThree = [1, 7, 3];
+const arrayFour = [1, undefined, 3, 5, -3];
+const arrayFive = [1, NaN, 3, 5, -3];
+
+function maxNumberOfArr(array) {
+    let maxNumber = array[0];
+
+    for (let i = 0; i <= array.length; i++) {
```
Comment (vladyslav-yermolin-moc) [inline homeworks/olena.ambitska_elena-ambitska/7-arrays-loops/arrays.js:10]:
```suggestion
    for (let i = 1; i <= array.length; i++) {
```

File: homeworks/olena.ambitska_elena-ambitska/7-arrays-loops/arrays.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+console.log(onlyPositiveNumbers(arrayOne));// [3, 1, 44, 3, 1, 2, 2, 1, 4]
+console.log(onlyPositiveNumbers(arrayTwo)); // [];
+console.log(onlyPositiveNumbers(arrayThree)); // [1, 7, 3];
+console.log(onlyPositiveNumbers(arrayFour)); // [1, 3, 5];
+console.log(onlyPositiveNumbers(arrayFive)); // [1, 3, 5];
+
+const customMap = (array, mapper) => {
+    const arr = [];
+
+    // eslint-disable-next-line no-restricted-syntax
```
Comment (vladyslav-yermolin-moc) [inline homeworks/olena.ambitska_elena-ambitska/7-arrays-loops/arrays.js:81]: Please use simple `for` loop and remove this

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: homeworks/olena.ambitska_elena-ambitska/7-arrays-loops/arrays.js (hunk 1 above)
Comment (elena-ambitska) [reply homeworks/olena.ambitska_elena-ambitska/7-arrays-loops/arrays.js]: @vladyslav-yermolin-moc done

File: homeworks/olena.ambitska_elena-ambitska/7-arrays-loops/arrays.js (hunk 2 above)
Comment (elena-ambitska) [reply homeworks/olena.ambitska_elena-ambitska/7-arrays-loops/arrays.js]: @vladyslav-yermolin-moc done

**Review by vladyslav-yermolin-moc — APPROVED** [review]

**Pull request merged**